docs/Makefile

```
@@ -1,67 +1,76 @@
 # Minimal makefile for Sphinx documentation
 #
 
-# You can set these variables from the command line, and also
-# from the environment for the first two.
+# You can set these variables from the command line or the environment
 SPHINXOPTS    ?=
 SPHINXBUILD   ?= sphinx-build
-SOURCEDIR     = .
-BUILDDIR      = _build
-VENV          = .sphinx/venv/bin/activate
+SOURCEDIR     ?= .
+BUILDDIR      ?= _build
+VENVDIR       ?= .sphinx/venv
+IN_VENV       ?= . $(VENVDIR)/bin/activate
 
 
 # Put it first so that "make" without argument is like "make help".
 help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	@echo "The following make targets are available:"
+	@echo "* install   -- set up a virtual-env to work on documentation"
+	@echo "* run       -- watch, build and serve the documentation"
+	@echo "* html      -- build HTML output under $(BUILDDIR)/html"
+	@echo "* serve     -- serve HTML output from $(BUILDDIR)/html"
+	@echo "* epub      -- build epub output under $(BUILDDIR)/epub"
+	@echo "* pdf       -- build pdf output under $(BUILDDIR)/pdf"
+	@echo "* clean-doc -- clean built doc files"
+	@echo "* clean     -- clean full environment"
```

todo: make target `distclean` is added, but not documented in the `help` target. 

Maybe we can add something like this:
```suggestion
	@echo "* clean     -- clean full environment"
	@echo "* distclean -- clean full environment and built doc files"
```

---

Yes, that was slightly deliberate as I only added it for the convenience of the packaging branch (i.e. it's not something I really intend end-users to know about or use given the `clean` and `clean-doc` targets already existed). I'm happy to document it if required, but we might want to keep it "semi-hidden" for the reasons given below...

---

That makes sense.